src/FSharp.Formatting.ApiDocs/GenerateHtml.fs

```diff
@@ -101,239 +105,232 @@ type HtmlRender(model: ApiDocModel, ?menuTemplateFolder: string) =
-
-                                              p [ Class "fsdocs-note" ] [ embed e ]
-
-                                          for e in m.Comment.Examples do
-                                              h5 [ Class "fsdocs-example-header" ] [ !! "Example" ]
-
-                                              p
-                                                  [ Class "fsdocs-example"
-                                                    if e.Id.IsSome then
-                                                        Id e.Id.Value ]
-                                                  [ embed e ]
-
-                                          //if m.IsObsolete then
-                                          //    obsoleteMessage m.ObsoleteMessage
-
-                                          //if not (String.IsNullOrEmpty(m.Details.FormatCompiledName)) then
-                                          //    p [] [!!"CompiledName: "; code [] [!!m.Details.FormatCompiledName]]
-                                          ] ] ] ] ]
+                                                        (match link with
+                                                         | None -> []
+                                                         | Some href -> [ a [ Href href ] [ !!nm ] ])
+                                                    td [] [ embed html ]
+                                                ]
+                                        ]
+
+                                    for e in m.Comment.Notes do
+                                        h5 [ Class "fsdocs-note-header" ] [ !! "Note" ]
+
+                                        p [ Class "fsdocs-note" ] [ embed e ]
+
+                                    for e in m.Comment.Examples do
+                                        h5 [ Class "fsdocs-example-header" ] [ !! "Example" ]
+
+                                        p [
+                                            Class "fsdocs-example"
+                                            if e.Id.IsSome then
+                                                Id e.Id.Value
+                                        ] [ embed e ]
+                                    //if m.IsObsolete then
+                                    //    obsoleteMessage m.ObsoleteMessage
+
+                                    //if not (String.IsNullOrEmpty(m.Details.FormatCompiledName)) then
+                                    //    p [] [!!"CompiledName: "; code [] [!!m.Details.FormatCompiledName]]
+                                    ]
+
+                              td
+                                  [ Class "fsdocs-member-xmldoc" ]
+                                  (match dtls, m.Comment.Summary.HtmlText.Trim() with
+                                   | [], _ -> [ smry ]
+                                   | _, "" -> dtls
+                                   | _, _ -> [ details [] ((summary [] [ smry ]) :: dtls) ])
```

Comment:
@nojaf  this is added since the session. It tries to handle missing summary and/or missing details; my solution was to only add the details tag when there's a "complete" XML comment with a summary and type info.

Reply:
This looks good, I slightly refactored it to use a list comprehension instead.
(Sorry, I couldn't help myself).